### PR TITLE
more content for more exmaples

### PR DESCRIPTION
**Pull request by Chief-Buka**
Added another example of how to easily create a neural net with custom layer names - using OrderedDict from collections module 

Added another example of how to save and load information (network, optimizer, loss, epochs) and how to load them back in to continue training

### CODE REVIEW

**Comment by li-phone** [comment]
你好，你的邮件已收到，稍后将会查看，尽快给你回复。
